src/Security/Authentication/test/TestExtensions.cs

```diff
@@ -77,4 +79,12 @@ public static Task DescribeAsync(this HttpResponse res, IEnumerable<Authenticati
         return res.Body.WriteAsync(xmlBytes, 0, xmlBytes.Length);
     }
 
+    public static IServiceCollection ConfigureAuthTestServices(this IServiceCollection services)
```

Comment:
nit: could drop Auth, ConfigureTestServices(), since this doesn't actually configure any auth services anyways.

Reply:
Maybe want to go with the same name between this one and GenerateServicesForTest for consistency too

Reply:
I'll do this in another PR to avoid having to rekick the build again. 😓